backend/src/main/java/com/redhat/cloud/notifications/db/repositories/BehaviorGroupRepository.java

```diff
@@ -41,6 +41,22 @@ public class BehaviorGroupRepository {
     @Inject
     OrgIdHelper orgIdHelper;
 
+    public BehaviorGroup createFull(String accountId, String orgId, @Valid BehaviorGroup behaviorGroup, List<UUID> endpoints, Set<UUID> eventTypes) {
+        BehaviorGroup saved = create(accountId, orgId, behaviorGroup);
+        if (endpoints != null) {
+            updateBehaviorGroupActions(accountId, orgId, saved.getId(), endpoints);
+        }
+
+        if (eventTypes != null) {
+            updateBehaviorEventTypes(accountId, orgId, saved.getId(), eventTypes);
+        }
+
+        entityManager.flush();
+        entityManager.refresh(saved);
```

Comment:
Could you add a comment here to explain why we're doing this?

Reply:
Updated